internal/runners/languages/languages.go

```diff
@@ -1,30 +1,47 @@
 package languages
 
 import (
-	"strings"
-
+	"github.com/ActiveState/cli/internal/analytics"
+	"github.com/ActiveState/cli/internal/config"
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/runbits/commitmediator"
+	"github.com/ActiveState/cli/internal/runbits/runtime"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
 	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/store"
+	"github.com/ActiveState/cli/pkg/platform/runtime/target"
 	"github.com/ActiveState/cli/pkg/project"
 )
 
 // Languages manages the listing execution context.
 type Languages struct {
-	out     output.Outputer
-	project *project.Project
+	out       output.Outputer
+	project   *project.Project
+	analytics analytics.Dispatcher
+	svcModel  *model.SvcModel
+	auth      *authentication.Auth
+	cfg       *config.Instance
 }
 
 // NewLanguages prepares a list execution context for use.
 func NewLanguages(prime primeable) *Languages {
 	return &Languages{
 		prime.Output(),
 		prime.Project(),
+		prime.Analytics(),
+		prime.SvcModel(),
+		prime.Auth(),
+		prime.Config(),
 	}
 }
 
+type languagePlainOutput struct {
```

Comment:
Note: cannot piggy-back off of `Language` struct because its `Version` and `Resolved` version fields must remain separate (e.g. "Auto" and "1.1.0") while this struct's `Version` field would be something like "Auto → 1.1.0".